src/citrine/resources/dataset.py

```diff
@@ -276,7 +275,7 @@ def delete_contents(
             *,
             timeout: float = 2 * 60,
             polling_delay: float = 1.0,
-            prompt_to_confirm: bool = None,
+            prompt_to_confirm: bool = True,
```

Comment:
Given that it previously defaulted to `None`, would it make more sense to make it default to `False` now? Or will users usually want to be prompted?

Reply:
Now that I think about it, the warning says that you will soon need to call the method with `prompt_to_confirm` explicitly set. So perhaps there shouldn't be a default